libs/utils/trace.py

```diff
@@ -629,14 +629,27 @@ def _sanitize_SchedEnergyDiff(self):
             'Suboptimal Reject')
 
     def _sanitize_SchedOverutilized(self):
-        """ Add a column with overutilized status duration. """
+        """
+        Add a column with overutilized status duration.
+        """
         if not self.hasEvents('sched_overutilized'):
             return
+
```

Comment:
I've moved it back into the previous commit, I can add a "beautifying" commit if you'd like.